kafka-rest/src/main/java/io/confluent/kafkarest/resources/v3/ProducerMetrics.java

```diff
@@ -237,26 +244,26 @@ private void addPercentiles(
 
   private MetricName getMetricName(String name, String doc) {
     return metrics.metricInstance(
-        new MetricNameTemplate(name, GROUP_NAME, doc, Collections.emptySet()), METRIC_TAGS);
+        new MetricNameTemplate(name, GROUP_NAME, doc, metricsTags.keySet()), metricsTags);
   }
 
-  void recordResponse() {
+  public void recordResponse() {
```

Comment:
Why did this and the others below become `public`?

Reply:
Left over from when we had the interface and various impl classes.  I thought I'd pulled these out already!